### DISCUSSION TITLE
Installing PWA through Microsoft Edge?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/winget-pkgs#37612] [Q&A]
Hello,

I have installed some sites manually but is it possible to use Winget for PWA installation through Microsoft Edge? I would like to create a PowerShell script with the software and PWA installation.

Thanks to you !

![2021-12-13 11_18_34-Window](https://user-images.githubusercontent.com/96052003/145794473-cb0e83c7-3d8d-4671-ab81-bf96798e40a7.png)

**Comment by OfficialEsco** [comment]
There is a issue for it: https://github.com/microsoft/winget-cli/issues/164
Schema version 1.0.0 and 1.1.0 does support PWA https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.1.0/manifest.installer.1.1.0.json#L49-L62
There is a Document for it https://github.com/microsoft/winget-cli/blob/master/doc/specs/%23164%20-%20PWA%20Support.md

However, i don't think the client got PWA support yet, also the Schema does not have everything required for a PWA, just like ZIP support.
